verifact/forum/schema.py

```diff
@@ -39,3 +40,87 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+# MUTATIONS
+
+
+class CreateQuestion(Mutation):
+    class Arguments:
+        created_at = DateTime()
+        text = String()
+        citation_url = String()
+        citation_title = String()
+        citation_image_url = String()
+
+    question = Field(QuestionNode)
+
+    def mutate(
+        self,
+        info,
+        text,
+        citation_url,
+        citation_title=None,
+        citation_image_url=None,
+        created_at=None,
+    ):
+        question = Question.objects.create(
+            created_at=created_at,
+            text=text,
+            citation_url=citation_url,
```

Comment:
I think we should validate that urls are in the correct format.

Reply:
using django's URLValidator - it requires the protocol but I think should be ok especially since most of the time the field will be populated by a user copy-pasting the URL anyways.